lib/bookingsync/api/client.rb

```diff
@@ -209,7 +214,7 @@ def paginate(path, options = {}, &block)
               first_request = false
             end
             break unless response.relations[:next]
-            response = response.relations[:next].get
+            response = response.relations[:next].public_send(request_method)
```

Comment:
i mean to have the call method on relation like before, and use that instead of using `public_send`?